src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -144,8 +147,17 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
     val promise = Promise[Seq[Try[Unit]]]()
 
     Future.sequence(batchStatements).onComplete {
-      case Success(_) => promise.complete(Success(Nil)) // Nil == all good
-      case Failure(e) => promise.failure(e)
+      case Success(_) =>
+        for (
+          p <- pubsub;
+          tag: String <- serialized.map(_.payload.map(_.tags).flatten).flatten.toSet
+        ) {
+          p ! DistributedPubSubMediator.Publish("akka.persistence.cassandra.journal.tag", tag)
```

Comment:
I'm actually having the thing off by default (by setting 0 seconds), since publishing messages is not useful unless consistent-read-delay is also zero. So you'd just change both settings together in an app to enable this feature.